source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -1316,9 +1316,7 @@ TEST(NiFakeServiceTests, NiFakeService_ParametersAreMultipleTypes_CallsParameter
           an_int_enum,
           a_float,
           expected_float_enum_value,
-          expected_string_size,
-          _))
-      .With(Args<8, 7>(ElementsAreArray(a_string, expected_string_size)))
```

Comment:
Note: These three tests were updated because the `ParametersAreMultipleTypes` signature changed in the base hapigen metadata to not include the `string size` input. So the expectation was updated accordingly.